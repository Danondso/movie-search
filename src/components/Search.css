.search-input {
    padding-top: 1rem;
}