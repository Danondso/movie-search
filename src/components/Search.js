import React from "react";
import "./Search.css";

const Search = ({ handleUserInputChange }) => {
  return (
    <div className="search-input">
      <input
        placeholder="movie-search..."
        onChange={handleUserInputChange}
      ></input>
    </div>
  );
};

export default Search;
