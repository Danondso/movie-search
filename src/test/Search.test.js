import React from 'react';
import Search from '../components/Search';
import renderer from 'react-test-renderer';

test('Search looks correct', () => {
    const component = renderer.create(
        <Search />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});