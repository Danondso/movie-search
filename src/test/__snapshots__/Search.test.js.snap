// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search looks correct 1`] = `
<div
  className="search-input"
>
  <input
    placeholder="movie-search..."
  />
</div>
`;
